### PR TITLE
Ezreal Skills now do damage!

### PR DESCRIPTION
**Pull request by DeudlyYT**
Ezreal Skills now do damage! (E not)

### CODE REVIEW

**Comment by DeudlyYT** [comment]
I created the pull request but dont merge it until R ult. Sorry and thanks

**Comment by MythicManiac** [comment]
This is alright for merging yet?

**Comment by DeudlyYT** [comment]
@MythicManiac All it's ok and works. But as someone said me to not merge every change and wait for have a lot of changes thats why I said to not merge it. But if you want you can do it.

Anyway I will try to do the Ult working but as it can take long you can merge this like part one.

File: GameServer/Content/Data/LeagueSandbox-Default/Champions/Ezreal/W.lua [hunk 1]
```diff
@@ -12,10 +12,11 @@ function onFinishCasting()
 end
 
 function applyEffects()
-	local attackDamage = getOwner():GetStats().AttackDamage
+
     local abilityPower = getOwner():GetStats().AbilityPower
-    local damage = getEffectValue(0) + attackDamage.Total + (0.4 * abilityPower.Total)
+    local damage = getEffectValue(0) + 0.8 * abilityPower.Total
```
Comment (DeudlyYT) [inline GameServer/Content/Data/LeagueSandbox-Default/Champions/Ezreal/W.lua]: This is the real formula

File: GameServer/Content/Data/LeagueSandbox-Default/Champions/Ezreal/W.lua [hunk 2]
```diff
@@ -12,10 +12,11 @@ function onFinishCasting()
 end
 
 function applyEffects()
-	local attackDamage = getOwner():GetStats().AttackDamage
```
Comment (DeudlyYT) [inline GameServer/Content/Data/LeagueSandbox-Default/Champions/Ezreal/W.lua]: Dont need this

**Comment by FurkanS1821** [comment]
W damages Ezreal himself, I just one shot myself lol

**Pull request merged**